odl/solvers/advanced/proximal_operators.py

```diff
@@ -1079,6 +1080,91 @@ def _call(self, x, out):
     return ProximalCConjKL
 
 
+def proximal_cconj_kl_version_two(space, lam=1, g=None):
```

Comment:
Would it make sense to speak of "first kind" and "second kind"? Alternatively, one could use the terms "data matching" and "prior matching" with some explanation.
However, I would probably keep the name of the existing KL divergence since it is somewhat the "default" for inverse problems, and use an additional tag for this slightly more exotic version as it is now.